examples/tutorials/hello-gltf/index.html

```diff
@@ -30,7 +30,7 @@
   // luma.registerDevices([WebGLDevice, WebGPUDevice]);
 
   const resolution = 2;
-  const device = luma.createDevice({width: resolution * window.innerWidth, height: resolution * window.innerHeight});
+  const device = luma.createDevice({width: resolution * window.innerWidth, height: resolution * window.innerHeight, type: 'webgl'});
```

Comment:
- Is there a problem with the way AnimationLoop is creating devices? 
- Maybe we can fix it at the source, rather than creating non-standard examples?
- When run as part of the website, the code in the HTML file doesn't execute, so please check the website too.

Reply:
https://github.com/visgl/luma.gl/pull/2191 & https://github.com/visgl/luma.gl/pull/2192